src/pages/CommonComponents/StarRating.js

```diff
@@ -79,3 +54,30 @@ export default class StarRating extends Component {
 }
 
 const STAR = [1, 2, 3, 4, 5];
+
+/*
+----------------------------
+How To Use
+----------------------------
+1. starRating 컴포넌트 POST,GET요청 보내는 함수는 starRating을 사용할 부모 컴포넌트에서 선언하여 props로 넘겨주세요
+
+2. 이미 존재하는 별점을 GET 받은 경우 starRatingForDetail 이름의 props로 넘겨주세요
+
+3. POST 요청 함수는 postStar 이라는 이름의 props로 넘겨주세요
+
+4. POST 요청 함수는 부모 컴포넌트 안에 다음과 같이 작성해주세요
+
+postStar = starRatingForPost => {
+  fetch(API_URLS.REVIEW, {
+    method: 'POST',
+    headers: {
+      Authorization: localStorage.getItem('TOKEN'),
+    },
+    body: JSON.stringify({
+      movie: this.props.id,
+      rating: starRatingForPost,
+    }),
+  }).then(this.props.updateRatingCount);
+};
+----------------------------
+*/
```

Comment:
이렇게 줄글로 써주면 정말 친절하고 좋겠지만 추후 나중에 문서화를 하게 될 때 이렇게 상세 설명을 적는 케이스보다는 type만 적어주고 간단한 설명정도 적어주는게 일반적입니다!

그리고 하위 컴포넌트에서 상위 컴포넌트의 로직에 너무 깊숙히 관여하는건 좋지 않습니다.
그렇게 되면 컴포넌트의 활용도, 확장성이 떨어져서 좋지 않습니다.

```suggestion

- starRatingForDetail(optional) : number
- 별점의 초기값

- postStar: function(nowRating)
  - parameter
     - nowRating: 현재 별점
- star가 onClick 될 때 호출될 함수, 호출 할 때 인자로 현재 별점을 전달해줍니다.

이정도로만 작성해줘도 충분해보이네요!
이부분은 수정하실 필요는 없고 보통 이런식으로 사용한다 정도로만 알아두세요!